iPetCare/Application/Dtos/Examinations/ExaminationsCreateExaminationDtoRequest.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace Application.Dtos.Examinations
+{
+    public class ExaminationsCreateExaminationDtoRequest
+    {
+        [Required]
+        public DateTime Date { get; set; }
+        public int ExaminationTypeId { get; set; }
+        public string NoteId { get; set; }
```

Comment:
NoteId i PetId jest Guid a nie stringiem (nie ma problemu z tym żeby przesłać to z frontu). Trzeba zmienić we wszystkch Response i Requestach (a także w kontrolerach jak mamy bez dtosów) stringi na Guid. Unikniemy dzięki temu zabawy z konwersją w serwisach.